pkg/controller/knativeserving/common/helpers.go

```diff
@@ -34,3 +41,93 @@ func UpdateConfigMap(cm *unstructured.Unstructured, data map[string]string, log
 		unstructured.SetNestedField(cm.Object, v, "data", k)
 	}
 }
+
```

Comment:
I don't love the name "helpers.go" for this file (my fault), and you've added so many related functions that I might lean toward putting them in their own say, "images.go" file? Still in the `common` package, of course.

Reply:
Done